Meta/gn/secondary/AK/BUILD.gn

```diff
@@ -11,7 +11,7 @@ shared_library("AK") {
   output_name = "ak"
 
   public_configs = [ ":ak_headers" ]
-  public_deps = [ ":ak_debug_gen" ]
+  public_deps = [ ":ak_debug_gen", ":ak_backtrace_gen" ]
```

Comment:
this one will fail the CI check because you didn't run `gn format` :)

the two commit hooks from the `pre-commit` yaml can help avoid CI yelling about linting.

<https://github.com/LadybirdBrowser/ladybird/blob/master/CONTRIBUTING.md#commit-hooks>